overviews/implement-api-bindings.md

```diff
@@ -171,5 +168,4 @@ foo
 
 ```
 
-Which produces: http://gateway.ipfs.io/ipfs/QmNtpA5TBNqHrKf3cLQ1AiUKXiE4JmUodbG5gXrajg8wdv
-
+which produces: http://gateway.ipfs.io/ipfs/QmNtpA5TBNqHrKf3cLQ1AiUKXiE4JmUodbG5gXrajg8wdv
```

Comment:
Because it's part of the sentence (far) above, that begins: "Basically, we end up with HTTP requests like this:" rather than a separate sentence. If we wanted a separate sentence, something like: "Such a set of requests produces: http://gateway..." would work.